airbyte-integrations/connectors/source-typeform/source_typeform/source.py

```diff
@@ -204,12 +204,16 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
             form_ids = config.get("form_ids", []).copy()
             auth = TokenAuthenticator(token=config["token"])
             # verify if form inputted by user is valid
-            for form in TrimForms(authenticator=auth, **config).read_records(sync_mode=SyncMode.full_refresh):
-                if form.get("id") in form_ids:
-                    form_ids.remove(form.get("id"))
-            if form_ids:
-                return False, f"Cannot find forms with IDs: {form_ids}. Please make sure they are valid form IDs and try again."
-            return True, None
+            for form in form_ids:
+                try:
+                    url = f"{TypeformStream.url_base}/forms/{form}/response"
+                    auth_headers = {"Authorization": f"Bearer {config['token']}"}
+                    session = requests.get(url, headers=auth_headers)
+                    session.raise_status()
+                except:
+                    return False, f"Cannot find forms with ID: {form}. Please make sure they are valid form IDs and try again."
+                return True, None
```

Comment:
```suggestion
            for form in form_ids:
                try:
                    url = f"{TypeformStream.url_base}/forms/{form}/response"
                    auth_headers = {"Authorization": f"Bearer {config['token']}"}
                    session = requests.get(url, headers=auth_headers)
                    session.raise_status()
                except:
                    return False, f"Cannot find forms with ID: {form}. Please make sure they are valid form IDs and try again."
            return True, None
```
The return must happens only after the for-loop is finished. Overall looks good.